src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(numb1) {
+    return function(numb2) {
+      count++;
+
+      if (count % 2 !== 0 || count === 2) {
+        return numb1 + numb2;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
Can you write code in `else` body outside `else`?

```suggestion
      }
      
      return 'Bzzz... Error!';
```